highfidelity/hifi
# Remove Limitless API

Replacement PR for https://github.com/highfidelity/hifi/pull/12837 (merged with latest master)

## QA Test Plan
- General smoke test.
- Open the Script Console (CTRL + ALT + J)     ,
   - type `LimitlessSpeechRecognition`
   - press Enter 
   - it should reply with:
```
[UncaughtException evaluate] Error: Can't find variable: LimitlessSpeechRecognition in about:console:1 [Backtrace] <global>() at about:console:1
ReferenceError: Can't find variable: LimitlessSpeechRecognition
```

Original notes from Triplelexx: "I have renamed a performance timer from "limitless" to "AnimDebugDraw" in the final commit, as that is handled in the scope."
